### DISCUSSION TITLE
I used the bridge from eth to sol and my eths arrived in nft format!

### DISCUSSION THREAD

**Discussion by Pooh7TR** [wormhole-foundation/wormhole#531] [Problems / bugs] [answered]
Hello good, I make this post since today I used the bridge from eth to solana and my eth arrived in nft format, I want to know what the solution can be since I looked everywhere and did not find anything at the moment

This is the wallet where the bridge was used: FDA4ybxSbHT9rhk6Ut3CbfKo7gcvuHBQzcGSJG2XGAUF, and ID: 2z3SnprhB79jCGs3VGDnqg5qQjUrVjrRYdG1mvgQqHPLUnk82c1xxzZ2JoVsKrEictbjhEJMNkB1Zuw2t9iB4zkN

https://gyazo.com/edbf7f10d86d762cf1a520269021bc71

**Comment by BLACKoin** [comment]
I have the same issue.

@Stevesol97 replied and suggested using dappnodeconnect  dot org to resolve the issue - IT's A SCAM! (per  official discord).

DO NOT USE THAT SITE

**Comment by leoluk** [comment, marked as answer]
What you describe is a UI bug in Phantom, which should've been fixed in the meantime.
